common/source/docs/common-holybro-kakuteh7-v2.rst

```diff
@@ -0,0 +1,246 @@
+                  GH plug)
+LED               WS2182 addressable LED signal wire
+Z-                Piezo buzzer negative leg
+=============     =================================================
+
+ESC Port 1
+----------
+
+=============     =================================================
+Pin               Function
+=============     =================================================
+B+                Battery positive voltage (2S-8S)
+R7                UART7 RX
+GND               Ground
+CURRENT           CURRENT
+M1                Motor signal output 1
+M2                Motor signal output 2
+M3                Motor signal output 3
+M4                Motor signal output 4
+=============     =================================================
+
+ESC Port 2
+----------
+
+=============     =================================================
+Pin               Function
+=============     =================================================
+B+                Battery positive voltage (2S-8S)
+R7                UART7 RX
+GND               Ground
+CURRENT           CURRENT
+M5                Motor signal output 5
+M6                Motor signal output 6
+M7                Motor signal output 7
+M8                Motor signal output 8
+=============     =================================================
+
+VTX Port
+--------
+=============     =================================================
+Pin               Function
+=============     =================================================
+Vtx+              9V for HD System or other VTX, by default ON/OFF is
+                  controlled by RELAY2. See :ref:`common-relay`
+G                 Ground
+T1                UART1 TX
+R1                UART1 RX
+G                 Ground
+R6                UART6 RX
+=============     =================================================
+
+UART Mapping
+============
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+   -  SERIAL0 -> USB
+   -  SERIAL1 -> UART1 (Telem1) DMA-enabled
+   -  SERIAL2 -> not available externally, internally connected to currently unsupported BT-LE module
+   -  SERIAL3 -> UART3 (SmartAudio VTX protocol by default)
+   -  SERIAL4 -> UART4 (GPS)D MA-enabled
+   -  SERIAL5 -> not available
+   -  SERIAL6 -> UART6 (Setup for serial RX Protocol SBUS by default) DMA-enabled
+   -  SERIAL7 -> UART7 (RX7 only available, normally used for ESC telemetry protocol by default)
+
+The SERIAL7 port (UART7) is normally for ESC telemetry, and has an R7 pin on
+both of the ESC connectors.
+
+Any UART may be re-tasked by changing its protocol parameter.
+
+Servo Output Mapping
+====================
+
+The PWM outputs are marked M1-M8 in the above pinouts. The corresponding servo outputs are:
+
+    - M1 -> Servo Output 4
+    - M2 -> Servo Output 1
+    - M3 -> Servo Output 2
+    - M4 -> Servo Output 3
+    - M5 -> Servo Output 5
```

Comment:
Is this correct?

Reply:
havent the foggiest....can you check...I did not create those tables...